docs/api/server/Environment-Variables.md

```diff
@@ -524,6 +527,27 @@ ONE_CONFIG_ENV=staging
 ONE_CONFIG_ENV=undefined
 ```
 
+## `ONE_CONFIG_USE_NATIVE_POLYFILL`
```

Comment:
```suggestion
## `ONE_CONFIG_USE_NATIVE_INTL`
```

I think this makes it a bit clearer what the variable does

Reply:
That's a good point, I have updated the name to this in [d76bbe5](https://github.com/americanexpress/one-app/pull/1258/commits/d76bbe516d1724bdd802457057f5727466429a71)